app/views/comments/create.html.erb

```diff
@@ -4,6 +4,5 @@ $('#answer-<%= @answer_id %>-comment-<%= @comment.id %>').show()
 $('#answer-<%= @answer_id %>-comment-count')[0].innerHTML = parseInt($('#answer-<%= @answer_id %>-comment-count')[0].innerHTML) + 1
 $('#answer-<%= @answer_id %>-comment-form textarea').val('');
 $('#answer-<%= @answer_id %>-comment-section .help-block').remove();
-$('textarea').val("");
```

Comment:
Hi! This is looking good, but will it still clear the comment textarea where you typed? Perhaps we need to leave this here, but to link it more closely to the correct textarea, like with `#answer-<%= @answer_id %>-comment-section textarea`?